pkg/kongstate/consumer_test.go

```diff
@@ -310,10 +310,25 @@ func TestConsumer_SetCredential(t *testing.T) {
 		{
 			name: "oauth2 without client_id",
 			args: args{
-				credType:   "oauth2",
-				consumer:   &Consumer{Consumer: kong.Consumer{Username: &username}},
-				credConfig: map[string]string{},
-				version:    standardVersion,
+				credType: "oauth2",
+				consumer: &Consumer{Consumer: kong.Consumer{Username: &username}},
+				credConfig: map[string]interface{}{
+					"client_id": "bar",
```

Comment:
The TC name "oauth2 without client_id", but with client_id configured. Which way we should proceed ?

Reply:
Copy-pasted and unchanged test names are our favorite passtime: e4b3e45d

Reply:
😆